test/jdk/java/lang/ProcessBuilder/Basic.java

```diff
@@ -77,6 +78,21 @@
 
     /* used for AIX only */
     static final String libpath = System.getenv("LIBPATH");
+    static final String expectedLibpath;
```

Comment:
Can you check the usage of libpath at line 1362. How is this case different from the other two at 1878 and 1943?
I would move the change to the `expected` value to move into the block that is already testing AIX.is at line 1367.

I would prefer to have a single static with the libpath expected by the 3 places it is used in the test.